server/webapp/static/backup_in_progress.html

```diff
@@ -1,6 +1,6 @@
 ﻿<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
or remove this entry?
https://github.com/gocd/gocd/blob/41cb67ad26a6c09dfeb26566ffa775b6f4c0b19b/server/webapp/WEB-INF/applicationContext-acegi-security.xml#L246